modules/core/include/opencv2/core/hal/hal.hpp

```diff
@@ -201,6 +201,9 @@ CV_EXPORTS void cvt32f16f( const float* src, float16_t* dst, int len );
 CV_EXPORTS void addRNGBias32f( float* arr, const float* scaleBiasPairs, int len );
 CV_EXPORTS void addRNGBias64f( double* arr, const double* scaleBiasPairs, int len );
 
+CV_EXPORTS void flip( int src_type, const uchar* src_data, size_t src_step, int src_width, int src_height, uchar* dst_data, size_t dst_step, int flip_mode );
+CV_EXPORTS void rotate( int src_type, const uchar* src_data, size_t src_step, int src_width, int src_height, uchar* dst_data, size_t dst_step, int angle );
```

Comment:
All functions from this file are defined somewhere in OpenCV codebase.
Added function are not implemented. So they should not be here.

Reply:
Lines removed in updated version of the patch
Thanks